### PR TITLE
#27 design and css

### PR DESCRIPTION
**Pull request by raphaelchpprt**
**Related issues**

#27

**Changes**

:heavy_plus_sign: edit design of authentication forms

:heavy_plus_sign:  edit design of home page 

:heavy_plus_sign:  edit design of new playlist page

:heavy_plus_sign:  edit design of show playlist page

:information_source: the design of the table tracks has not been done yet

* * *
- [x] I ran this code locally

### CODE REVIEW

File: src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -69,34 +72,22 @@ const Navbar = () => {
...
             </Menu.Item>
           )}
 
           {logStatus ? (
-            <Menu.Item key="9" disabled icon={<SettingOutlined />}>
-              <Button type="button" onClick={disconnection}>
-                Disconnect
-              </Button>
+            <Menu.Item key="9" icon={<LogoutOutlined />}>
+              <NavLink type="button" onClick={disconnection} to="/">
```
Comment (floriansr) [inline src/components/Navbar/index.jsx:82]: cool

File: src/components/Navbar/index.jsx [hunk 2]
```diff
@@ -69,34 +72,22 @@ const Navbar = () => {
...
+                Logout
+              </NavLink>
             </Menu.Item>
           ) : (
             <Menu.Item key="1" icon={<UserOutlined />}>
-              <NavLink to="/login" activeClassName="active">
-                Connection
-              </NavLink>
+              <NavLink to="/login">Sign in</NavLink>
```
Comment (floriansr) [inline src/components/Navbar/index.jsx:88]: why did you delete the activeClassName attibute ?

File: src/components/Navbar/index.jsx [hunk 3]
```diff
@@ -69,34 +72,22 @@ const Navbar = () => {
...
             </Menu.Item>
           )}
-
-          <Menu.Item key="10">
```
Comment (floriansr) [inline src/components/Navbar/index.jsx:91]: It was fun, bye friend 🤝

File: src/components/Navbar/index.jsx (hunk 2 above)
Comment (raphaelchpprt) [reply src/components/Navbar/index.jsx]: Readded :)

File: src/services/SpotifyAPIManager/index.js [hunk 4]
```diff
@@ -21,15 +21,17 @@ const API = axios.create({
...
-  ({
-    headers,
-    ...config
-  }) => ({
+  ({ headers, ...config }) => ({
     ...config,
     headers: {
       ...headers,
       'Content-Type': 'application/json',
+<<<<<<< HEAD
```
Comment (floriansr) [inline src/services/SpotifyAPIManager/index.js]: To be erase

File: src/services/SpotifyAPIManager/index.js [hunk 5]
```diff
@@ -21,15 +21,17 @@ const API = axios.create({
...
       Authorization: `Bearer ${token}`,
+=======
+      Authorization:
```
[the remark this answers is not in the document]
Comment (floriansr) [reply src/services/SpotifyAPIManager/index.js]: resolved
